File: PersonasVB/Contacto.vb
Public Class Contacto
    Inherits Persona
    Private telefono_ As String
    Private email_ As String

    Public Property Telefono
        Get
            Return telefono_

        End Get

        Set
            telefono_ = Value
        End Set
    End Property

    Public Property Email
        Get
            Return email_
        End Get

        Set
            email_ = Value
        End Set

    End Property

    Public Sub New()
        telefono_ = ""
        email_ = ""
        ApPaterno_ = ""
        ApMaterno_ = ""
        nombre_ = ""
        Nacimiento_ = DateTime.Now
    End Sub
    Public Sub New(tel As String, correo As String, nombres As String, ApePaterno As String, ApeMaterno As String, FechaNacimiento As DateTime)
        telefono_ = tel
        email_ = correo
        ApPaterno_ = ApePaterno
        ApMaterno_ = ApeMaterno
        nombre_ = nombres
        Nacimiento_ = FechaNacimiento
    End Sub

    Public Overloads Function toString() As String
        Return nombre_ + " " + ApPaterno_ + " " + ApMaterno_ + " " + Nacimiento_ + "," + telefono_ + ", " + email_
    End Function


End Class

File: PERSONAS/Contacto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PERSONAS
{
    public class Contacto : Personas
    {
        private string telefono;
        private string email;

        public string Telefono {
            get { return telefono; }
            set { telefono = value; }
        }
        public string Email {
            get { return email; }
            set { email = value; }
        }
        public Contacto() {
            telefono = "";
            email = "";
            ApPaterno = "";
            ApMaterno = "";
            nombre = "";
            Nacimiento = DateTime.Now;
        }
        public Contacto(string telefono, string email, string nombre, string ApellidoPaterno, string ApellidoMaterno, DateTime FechaNacimiento)
        {
            this.telefono = telefono;
            this.email = email;
            this.nombre = nombre;
            this.ApMaterno = ApellidoMaterno;
            this.ApPaterno = ApellidoPaterno;
            this.Nacimiento = FechaNacimiento;
        }
        public string toString(){
            return nombre + " "+ApPaterno+" "+ApMaterno+" "+Nacimiento+","+telefono+", "+email;
}
    }
}
